core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineStrategy.java

```diff
@@ -296,4 +303,16 @@ default void setConnectionToReadOnly(Connection connection, boolean readOnly)
    */
   default void throwIfCrossPartitionScanOrderingOnBlobColumnNotSupported(
       ScanAll scanAll, TableMetadata metadata) {}
+
+  /**
+   * Returns whether type conversion is supported between the two data types in the underlying
+   * storage.
+   *
+   * @param from the source data type
+   * @param to the target data type
+   * @return true if type conversion is supported, false otherwise
+   */
+  default boolean isTypeConversionSupportedInternal(DataType from, DataType to) {
```

Comment:
Not a big deal, but I feel this is probably more natural. (or Internally can be removed)
```suggestion
  default boolean isTypeConversionSupportedInternally(DataType from, DataType to) {
```

Reply:
I simply removed `Internal` in [3f9e399](https://github.com/scalar-labs/scalardb/pull/3028/commits/3f9e399e8ca3f99165302a03db883a31b2428db0).